tests/Zend/ContainerTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
+
+namespace Northwoods\Container\Zend;
+
+use Auryn\Injector;
+use Northwoods\Container\InjectorConfig;
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Zend\ContainerConfigTest\AbstractExpressiveContainerConfigTest;
+use Zend\ContainerConfigTest\SharedTestTrait;
```

Comment:
Do you support shared/non-shared services as well? If so you should use that trait in your test class, otherwise is useless and can be removed.

Reply:
There is no support for share flagging at this time. I'll remove it.